site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -78,60 +98,79 @@ def __init__(self, inspire_metadata):
 
     def _transform_titles(self):
         """Mapping of INSPIRE titles to metadata.title and additional_titles."""
+        current_app.logger.info("Start transforming titles.")
         inspire_titles = self.inspire_metadata.get("titles")
+        current_app.logger.info(f"Titles from INSPIRE: {inspire_titles}.")
```

Comment:
IMO too detailed logging
all the transformations we could log at the end of the transform, in debug mode, just reporting the full transformed dict